lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -81,6 +83,36 @@ class DetailsRenderer {
     element.appendChild(items);
     return element;
   }
+
+  /**
+   * @param {!DetailsJSON} list
+   * @return {!Element}
+   */
+  _renderCards(list) {
+    const element = this._dom.createElement('details', 'lighthouse-details');
+    if (list.header) {
+      element.appendChild(this._dom.createElement('summary')).textContent = list.header.text;
```

Comment:
> You mean just use a string: `header: 'View details'`?

or whatever. I'm just wondering why it's a full `DetailsJSON` if we don't use it